paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -887,33 +887,47 @@ class PaparazziPluginTest {
     firstResourceFile.copyTo(destResourceFile, overwrite = false)
 
     val firstRun = gradleRunner
-      .withArguments(":preparePaparazziDebugResources", "--build-cache", "--stacktrace")
+      .withArguments("testDebug", "--build-cache", "--stacktrace")
       .forwardOutput()
       .runFixture(fixtureRoot) { build() }
 
     with(firstRun.task(":preparePaparazziDebugResources")) {
       assertThat(this).isNotNull()
       assertThat(this!!.outcome).isEqualTo(SUCCESS)
     }
+    with(firstRun.task(":testDebugUnitTest")) {
+      assertThat(this).isNotNull()
+      assertThat(this!!.outcome).isEqualTo(SUCCESS)
+    }
 
     val resourcesFile = File(fixtureRoot, "build/intermediates/paparazzi/debug/resources.json")
 
+    println("JROD (run 1): ${resourcesFile.readText()}")
     var config = resourcesFile.loadConfig()
     assertThat(config.projectResourceDirs).containsExactly("src/main/res", "src/debug/res")
 
+    assertThat(resourcesFile.exists()).isTrue()
     buildDir.deleteRecursively()
+    assertThat(resourcesFile.exists()).isFalse()
```

Comment:
Looks like this assertion fails on Windows

Reply:
ah, looks like we're holding onto the File handle after loading the json, which is preventing the parent buildDir folder from being cleaned up, i.e., 
```
  private fun File.loadConfig() = CONFIG_ADAPTER.fromJson(source().buffer())!!
```
vs
```
  private fun File.loadConfig() = source().buffer().use { CONFIG_ADAPTER.fromJson(it)!! }
```

Interesting that this is failing on Windows only, though.